deploy/k8s/base/nginx/deployment.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: nginx
+  template:
+    metadata:
+      labels:
+        component: nginx
+    spec:
+      initContainers:
+        - name: labelstudio-static-copy
+          image: heartexlabs/label-studio:1.0.2
+          command:
+            - sh
+            - "-ce"
+            - |
+              set -ex
+              cp -r /label-studio/label_studio/* /tmp/static/
```

Comment:
Hey! Sorry for such a late response.
This step is no longer necessary because I've introduced well-known entrypoint.d init scheme: https://github.com/heartexlabs/label-studio/pull/1363
So now all that you have to do - just pass two variables(https://github.com/heartexlabs/label-studio/pull/1397):
```
LABEL_STUDIO_COPY_STATIC_DATA=true
LABEL_STUDIO_STATIC_DATA_TARGET=<your_volume>
```

Reply:
Ha great, this was clearly the tricky part of the port to k8s! :)
I'm not completely sure how to go though, just remove this whole "labelstudio-static-copy" initContainer here, and setup the two env variables within `deploy/k8s/overlays/prod/env.yml ` ?

Reply:
- Completely remove initContainer from nginx
- Add k8s readinessProbe to deploy/k8s/base/labelstudio/deployment.yml (as a replacement for willwill/wait-for-it)
- Add k8s nginx readinessProbe to deploy/k8s/base/nginx/deployment.yml (an example how to implement readiness endpoint on nginx side https://github.com/nginxinc/kubernetes-ingress/issues/90#issuecomment-266451709)

Reply:
I'm sorry but it feels like you know k8s way better than I do, I need to browse through the documentation for most of your suggestions. I feel like this would be easier if you would take over this pull request and make those changes properly rather than me trying and errorring, what do you think? (I don't really care for authorship honestly, was just trying to share with the community a solution that worked for our case)